website/docs/examples/controller_header/README.md

```diff
@@ -0,0 +1,165 @@
+The end point of the learned track can seamlessly become the starting point, provided the robot is
+oriented towards its recorded initial position.
+This allows the robot to navigate environments bidirectionally, offering greater autonomy and
+adaptability in dynamic settings.
+
+2. **Initiating from Intermediate Points**:
+You don't always have to start the robot from the beginning of a track.
+The controller is smart enough to allow track following from any intermediate point within the path.
+This is especially useful if the robot needs to start its journey from a point that's not the
+traditional beginning or end.
+:::
+
+:::info Requirements for Following a Track:
+For the controller to begin guiding the robot along a track, certain conditions must be met to
+ensure accurate and safe navigation:
+
+**Proximity to the Track**: The robot must be situated within a maximum distance of 0.1 meters from
+any existing point on the track.
+This ensures it's close enough to align itself correctly and follow the track.
+
+**Orientation or Heading**: The robot's heading needs to align closely with the heading of the nearest
+point on the track.
+Specifically, it should be within 30 degrees of the track point's heading.
+This condition ensures the robot is oriented correctly and can safely navigate along the desired
+path without sudden, unexpected maneuvers.
+:::
+
+## Tracks
+
+A track is essentially a sequence of waypoints, guiding the robot from its starting point,
+through various intermediary points, to its destination.
+Each waypoint in this track provides specific spatial information about where the robot should
+be and how it should be oriented at that point.
+
+### Define a Track
+
+A track is represented as a JSON file, where each waypoint is defined by a pose.
+A pose captures the robot's position and orientation in the world.
+Here's an example of what a single pose looks like:
+
+```bash
+   {
+      "pose": {
+        "aFromB": {
+          "rotation": {
+            "unitQuaternion": {
+              "imag": {
+                "z": -0.9932486190247055
+              },
+              "real": 0.11600508956729176
+            }
+          },
+          "translation": {
+            "x": 2674.593868581748,
+            "y": 4297.325262829638
+          }
+        },
+        "frameA": "world",
+        "frameB": "robot",
+        "tangentOfBInA": {
+          "linearVelocity": {
+            "x": 0.4467579546503179
+          },
+          "angularVelocity": {
+            "z": 0.00475509022854358
+          }
+        }
+      },
+      "heading": -2.909058930621276
+    },
+```
+
+### Examples
+
+Now that you have a good understanding of how the controller works, try some of the
+controller examples:
+
+- [Record a Track](/docs/examples/record_track)
+- [Follow a Track](/docs/examples/controller_track)
+
+:::warning Danger
+**WARNING**: The controller examples will cause the Amiga to drive when the dashboard is in auto mode.
```

Comment:
I think it's appropriate to have a warning tip at the top of examples/tutorials that will cause the robot to move, but it doesn't need to be as red and scary. How about:
* Move this warning to the top of "Record Track" (matching "Follow a Track").
* Change formatting to `:::caution` in both places.

Reply:
#214